ideascube/configuration/errors.py

```diff
@@ -0,0 +1,21 @@
+class InvalidConfigurationValue(Exception):
```

Comment:
In the last push to this branch I renamed the file `exceptions.py` and added `Error` to the end of the class names.